src/WebJobs.Script.WebHost/Security/KeyManagement/SecretManager.cs

```diff
@@ -76,9 +77,9 @@ protected virtual void Dispose(bool disposing)
 
         public async virtual Task<HostSecretsInfo> GetHostSecretsAsync()
         {
-            using (_metricsLogger.LatencyEvent(GetMetricEventName(MetricEventNames.SecretManagerGetHostSecrets)))
```

Comment:
Here and below, saving two metrics log calls. We only need to be doing this when we're loading and want to capture the latency